Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG NODE_VERSION=14.17.5
```

Comment:
that actually should not be matter, since the server is self contained

Reply:
Oh right then, should we just leave it at latest?

Reply:
something like  ubuntu:18.04 will be more appropriate for real life usage